Revert-tvOS/Resources/Data/MapLocations.plist

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We shouldn't be duplicating these assets for the tvOS target if they can be shared with the iOS side of things.